status.go

```diff
@@ -87,20 +88,20 @@ func StatusFormat() []byte {
 		buf.WriteString(" not found\n")
 	} else {
 		buf.WriteString("\n")
-		fmt.Fprintf(buf, " Num  Device              Vndr:Prod  Model\n")
+		fmt.Fprintf(buf, " Num  Device              Vndr:Prod  Port  Model\n")
 		for i, status := range devs {
 			info, _ := status.desc.GetUsbDeviceInfo()
 
-			fmt.Fprintf(buf, " %3d. %s  %4.4x:%.4x  %q\n",
+			fmt.Fprintf(buf, " %3d. %s  %4.4x:%.4x  %d %-32q",
```

Comment:
Please, don't remove the trailing \n

Otherwise, resulting string is too wide and it is not comfortable to read the status at the standard 80-columns terminal

Reply:
Sure thing, the STATUS value is no on the next line again.